public/app/types/serviceaccount.ts

```diff
@@ -0,0 +1,96 @@
+import { OrgRole, Unit } from '.';
+import { SelectableValue } from '@grafana/data';
+
+export interface OrgServiceAccount {
+  avatarUrl: string;
+  email: string;
+  lastSeenAt: string;
+  lastSeenAtAge: string;
+  login: string;
+  name: string;
+  orgId: number;
+  role: OrgRole;
+  serviceAccountId: number;
+}
+
+export interface ServiceAccount {
+  id: number;
+  label: string;
+  avatarUrl: string;
+  login: string;
+  email: string;
+  name: string;
+  orgId?: number;
+}
+
+export interface ServiceAccountDTO {
+  id: number;
+  login: string;
+  email: string;
+  name: string;
+  isGrafanaAdmin: boolean;
+  isDisabled: boolean;
+  isAdmin?: boolean;
+  isExternal?: boolean;
+  updatedAt?: string;
+  authLabels?: string[];
+  theme?: string;
+  avatarUrl?: string;
+  orgId?: number;
+  lastSeenAtAge?: string;
+  licensedRole?: string;
+  permissions?: string[];
+  teams?: Unit[];
+  orgs?: Unit[];
+}
+
+export interface ServiceAccountsState {
+  serviceAccounts: OrgServiceAccount[];
+  searchQuery: string;
+  searchPage: number;
+  isLoading: boolean;
+}
+
+export interface ServiceAccountSession {
+  id: number;
+  createdAt: string;
+  clientIp: string;
+  isActive: boolean;
+  seenAt: string;
+  browser: string;
+  browserVersion: string;
+  os: string;
+  osVersion: string;
+  device: string;
```

Comment:
question: how many of these apply to service accounts?

Reply:
this can for sure be removed